### DISCUSSION TITLE
Busking 20240330

### DISCUSSION THREAD

**Discussion by Sean880128** [J-tour/J-tour.github.io#24] [General]
#20 reference
# Info
Replacement for Busking 20240330
_Date_ 2024-03-30 (Sat.)
## Cause
日程在即，目前各單位回覆如下：

- 文化局尚未回覆
- 影城回覆「沒有在做這一塊」
- 其他場地須要10天以上的書面申請流程

## Action plan
![image](https://github.com/J-tour/J-tour.github.io/assets/163014964/378590f1-8636-41cb-8b28-97fcc8d801fa)

> [新勢公園](https://maps.app.goo.gl/Mob3h8rJCrU12ttx8)練團方案

1. 新勢公園練團，不對外表演收費：攜帶器材圍圈練團，練習並配置固定常備歌編曲
2. 新勢公園僥倖表演：依照前例在新勢公園表演，遭驅趕再轉移陣地或去 @Sean880128 舅舅烘豆廠喝咖啡練團
3. 桃園一日遊：不表演了，直接來去桃園玩一波🤘🏻

![image](https://github.com/J-tour/J-tour.github.io/assets/163014964/17ae5c21-7eef-4cc6-9fb9-aa29d372e25e)

> 新勢公園場地寬廣，不須擔心違反噪音管制法

**Comment by Cheng-ChiAn** [comment]
支持2或3

**Comment by Sean880128** [comment]
我all ok

**Comment by DipsyKoo** [comment]
我都可以~

**Comment by ShengHaoLo** [comment]
我偏好在新勢公園表演看看，不過我當天下午表演完就要先離開了，晚上還有其他活動

**Reply by ShengHaoLo** [reply]
@Sean880128 記得決定好後將AP (action plan)轉成AR (action report)，再transfer to issue

**Reply by ShengHaoLo** [reply]
@Sean880128 請填report結案，並轉會計登帳
@xinyannn1 請紀錄當天收支
感謝
